### PR TITLE
Feat/default account

### PR DESCRIPTION
**Pull request by volotovskyy**

### CODE REVIEW

File: spec/paths/default-account@v1@integrations@salary.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    - Default-account  
+  responses:
+    200:
+      description: An array of integration modules
+      content:
+        application/json:
+          schema:
+            type: array
+            items:
+              $ref: "#/components/schemas/IntegrationModule"
```
Comment (sebastianjonasson) [inline spec/paths/default-account@v1@integrations@salary.yaml:15]: Invalid path. Typo in filename: `IntegraionModule.yaml`

Comment (volotovskyy) [reply spec/paths/default-account@v1@integrations@salary.yaml:15]: thanks for a catch.
fixed

File: spec/components/schemas/IntegrationModule.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+properties:
+  id:
+    type: string
+    format: ObjectID
+  module:
+    type: string
+    description: Name of the integration
+  image:
+    type: string
+    description: Link to the integration partner logo
```
Comment (jheander) [inline spec/components/schemas/IntegrationModule.yaml:11]: Maybe add a `type` parameter to specify if it is an invoice or salary integration?

File: spec/paths/default-account@v1@integrations@invoice.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+get:
+  summary: Invoice integrations
+  description: |
+    Get possible integrations options for default setup
+  tags:
```
Comment (jheander) [inline spec/paths/default-account@v1@integrations@invoice.yaml:5]: Maybe use "Sign up" or "Company" as tag? We can document more APIs in that section then.

File: spec/components/schemas/IntegrationModule.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+type: object
+properties:
+  id:
+    type: string
+    format: ObjectID
```
Comment (jheander) [inline spec/components/schemas/IntegrationModule.yaml:5]: Maybe `title` or `name` is more consistent with our other APIs for the name of the integration.

File: spec/components/schemas/IntegrationModule.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+type: object
+properties:
+  id:
+    type: string
```
Comment (jheander) [inline spec/components/schemas/IntegrationModule.yaml:4]: Why does the id have to be an ObjectId? We are not storing the integrations in a mongo database anyway. Would be nice to just have them called like `fortnox-payroll` or something.

**Pull request merged**